include/h2o.h

```diff
@@ -2125,7 +2128,7 @@ static inline void h2o_doublebuffer_dispose(h2o_doublebuffer_t *db)
 
 static inline h2o_iovec_t h2o_doublebuffer_prepare(h2o_doublebuffer_t *db, h2o_buffer_t **receiving, size_t max_bytes)
 {
-    assert(db->bytes_inflight == 0);
+    assert(db->inflight == 0);
```

Comment:
By convention, we omit equality operators when testing a boolean value. We generally do things like `if (db->inflight)`, `assert(!db->inflgiht)`.

Would you mind adjusting the code that deals with `inflight` to follow the practice?